tweaks/sidebar.css

```diff
@@ -33,6 +33,16 @@
   opacity: 1%;
 }
 
+/* Add a pseudoelement to increase the hover area for the sidebar */
+
+#sidebar-box::before {
+  content: '';
+  position: absolute;
+  width: 25px;
```

Comment:
change the value to 16px so that it doesn't trigger when moving on the edge of the website
apart from that lgtm